src/api/index.js

```diff
@@ -331,3 +331,14 @@ export function getInstallProperties() {
   const uri = `${apiRoot}/v1/properties`;
   return get(uri);
 }
+
+export function shouldDisplayLogout() {
+  console.log('toad skin');
+  const routesUri = getResourcesAPI({
+    group: 'route.openshift.io',
+    version: 'v1'
+  });
+  const thePromise = get(routesUri, { Accept: 'text/plain' });
+  console.log(`ok: ${thePromise}`);
```

Comment:
let's remove this, and we can then `return get(...)`

Reply:
😆 yeah was also for debugging, gone, thanks